### PR TITLE
Fixes: #1594 - Enable FIPS

### PR DESCRIPTION
**Pull request by ebenahar**
Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

**Comment by clacroix12** [comment]
Please add `Fixes: #1594` to description but otherwise LGTM

File: conf/ocsci/fips.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+ENV_DATA:
+  fips: 'true'
```
Comment (petr-balogh) [inline conf/ocsci/fips.yaml]:
```suggestion
  fips: true
```
use boolean value and not string

File: ocs_ci/templates/ocp-deployment/install-config-aws-ipi.yaml.j2 [hunk 2]
```diff
@@ -35,6 +35,9 @@ networking:
   networkType: OpenShiftSDN
   serviceNetwork:
     - 172.30.0.0/16
+{% if fips %}
+fips: {{ fips | default('false') }}
```
Comment (petr-balogh) [inline ocs_ci/templates/ocp-deployment/install-config-aws-ipi.yaml.j2]:
```suggestion
fips: {{ fips }}
```
no need for default value as if not defined we don't  add anything

File: conf/ocsci/fips.yaml (hunk 1 above)
Comment (ebenahar) [reply conf/ocsci/fips.yaml]: this is what I tried before and ended up the upper case with fips: True 
See the previous commit

File: ocs_ci/templates/ocp-deployment/install-config-aws-ipi.yaml.j2 (hunk 2 above)
Comment (ebenahar) [reply ocs_ci/templates/ocp-deployment/install-config-aws-ipi.yaml.j2]: ack. Will fix

**Comment by ebenahar** [comment]
Can't verify with FIPS enabled due to https://bugzilla.redhat.com/show_bug.cgi?id=1807225. The deployment with FIPS disabled passes as seen above

**Pull request merged**